packages/web-config-server/src/apiV1/dataBuilders/generic/percentage/percentagesOfValueCounts.js

```diff
@@ -138,10 +138,17 @@ export class PercentagesOfValueCountsBuilder extends DataBuilder {
 
     const allResults = numeratorResults;
 
+    const numeratorResultsMap = {};
+    numeratorResults.forEach(analytic => {
+      const { organisationUnit, dataElement, value, period } = analytic;
+      numeratorResultsMap[`${organisationUnit}|${dataElement}|${value}|${period}`] = analytic;
+    });
```

Comment:
Small, but wouldn't mind this being pushed into a function, eg.
```
const getResultMapKey = (organisationUnit, dataElement, value, period) => `${organisationUnit}|${dataElement}|${value}|${period}`;
```
Would make it a little easier to read and less prone to human error 😄